### PR TITLE
fix(*): Regenerates protobuf files for protobuf 3.2

### PR DESCRIPTION
**Pull request by thomastaylor312**
Protobuf 3.2 was recently released and generates a slightly different file from the protobuf definitions. These were all changes to the autogenerated byte array.

### CODE REVIEW

**Comment by thomastaylor312** [comment]
No idea why my CLA isn't showing up

**Comment by thomastaylor312** [comment]
I signed it!

**Comment by thomastaylor312** [comment]
I signed it!

File: pkg/proto/hapi/release/status.pb.go [hunk 1]
```diff
@@ -84,23 +84,24 @@ func init() {
...
-	0xa6, 0x1e, 0x43, 0x70, 0x96, 0x3c, 0xb9, 0xd6, 0x83, 0xf3, 0xde, 0x3b, 0x0a, 0x6c, 0xec, 0xe6,
-	0x87, 0xee, 0x7e, 0x03, 0x00, 0x00, 0xff, 0xff, 0xd4, 0x11, 0x21, 0x30, 0x86, 0x01, 0x00, 0x00,
+	// 291 bytes of a gzipped FileDescriptorProto
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x09, 0x6e, 0x88, 0x02, 0xff, 0x54, 0x90, 0xdf, 0x6a, 0xc2, 0x30,
+	0x14, 0xc6, 0x57, 0xad, 0x3a, 0x8f, 0x22, 0x21, 0x1b, 0xac, 0xca, 0x06, 0xc5, 0xab, 0xde, 0xac,
+	0x05, 0xf7, 0x04, 0xdb, 0x12, 0x87, 0xac, 0x54, 0x69, 0x2b, 0xfb, 0x73, 0x53, 0xaa, 0x9e, 0x39,
+	0xa1, 0x34, 0xd2, 0x24, 0x17, 0x7b, 0x88, 0xbd, 0xf3, 0x68, 0x2b, 0x74, 0x5e, 0x7e, 0xf9, 0xfd,
+	0x4e, 0xce, 0xc7, 0x81, 0xf1, 0x77, 0x7a, 0x3c, 0x78, 0x05, 0x66, 0x98, 0x4a, 0xf4, 0xa4, 0x4a,
+	0x95, 0x96, 0xee, 0xb1, 0x10, 0x4a, 0xd0, 0x61, 0x89, 0xdc, 0x13, 0x9a, 0xdc, 0x9d, 0x89, 0x0a,
+	0xa5, 0x4a, 0xa4, 0x3e, 0x28, 0xac, 0xe5, 0xc9, 0x78, 0x2f, 0xc4, 0x3e, 0x43, 0xaf, 0x4a, 0x1b,
```
Comment (technosophos) [inline pkg/proto/hapi/release/status.pb.go]: Shouldn't the sixth value in be `0x7c`? Just kidding.

**Review by technosophos — APPROVED** [review]

**Pull request merged**